session-hovercard-component.js

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright (c) 2015-2016 Digital Bazaar, Inc. All rights reserved.
+ */
+define([], function() {
+
+'use strict';
+
+function register(module) {
+  module.component('brSessionHovercard', {
+    bindings: {
+      label: '@brLabel',
+      onSignOut: '&brOnSignOut',
+      onRefresh: '&brOnRefresh'
+    },
+    controller: Ctrl,
+    templateUrl: requirejs.toUrl(
+      'bedrock-angular-session/session-hovercard-component.html'),
+    transclude: {
+      'action-slot': "?brSessionHovercardActionSlot",
+      'identity-slot': "?brSessionHovercardIdentitySlot",
+      'modal-slot': "?brSessionHovercardModalSlot"
```

Comment:
We're not using a "slot" naming convention so, at least for the public interface, we should drop the "Slot" suffix. Also, I don't think we should have an "Identity" slot (or an "Action" slot) at all -- it's too prescriptive for how a hovercard should be used. I think it should just be `brSessionHovercardContent`. I don't understand the modal slot either ... can't whomever is providing the content just include a modal if they want? What's the extra slot for?